### PR TITLE
Add documentation for auto-switching theme

### PR DESCRIPTION
**Pull request by Kuppjaerk**
Added documentation regarding auto-switching themes to the default settings file, according to ([#9627](https://github.com/zed-industries/zed/issues/9627)).

Release Notes:

- N/A

### CODE REVIEW

File: assets/settings/default.json [hunk 1]
```diff
@@ -1,6 +1,18 @@
 {
   // The name of the Zed theme to use for the UI
   "theme": "One Dark",
+  // The theme can also be set to follow system preferences.
```
Comment (maxdeviant) [inline assets/settings/default.json:4]: Could we move this comment up above the `theme` field?

**Review by maxdeviant — CHANGES_REQUESTED** [review]

File: assets/settings/default.json (hunk 1 above)
Comment (Kuppjaerk) [reply assets/settings/default.json:3]: Absolutely! Is it okay if this is in a seperate commit?

Comment (maxdeviant) [reply assets/settings/default.json:3]: > Absolutely! Is it okay if this is in a seperate commit?

Yep, that's fine.

We squash commits when merging to `main`, so it all ends up in a single commit anyways 🙂

Comment (Kuppjaerk) [reply assets/settings/default.json:3]: Ok, great! I've submitted a new commit for this, do I need to do anything more in this PR to indicate that the change is made? I'm not too familiar with submitting PR:s here :)

**Review by maxdeviant — APPROVED** [review]
Thanks!

**Pull request merged**